### ISSUE TITLE
Question regarding loss scaling

### ISSUE THREAD

**Issue by gouthamk1998** [ultralytics/yolov5#481]
## ❔Is there a particular reason for scaling the loss by `bs` in `compute_loss` function.?? (`return loss * bs, torch.cat((lbox, lobj, lcls, loss)).detach()`)

Because of that loss used for gradient computations is `loss*bs` while the loss reported in screen is different(unscaled version)

**Comment by glenn-jocher** [comment]
@gouthamk1998 yes correct, otherwise different models and batch sizes would show different losses, making comparisons impossible.
